analysis/report_templates/default.html

```diff
@@ -100,6 +100,24 @@ <h2 class="green-text text-darken-3">experiment summary</h2>
 
             <h2 class="green-text text-darken-3">{{ benchmark.name }} summary</h2>
 
+            {% if benchmark.fuzzers_with_not_enough_samples %}
+            <div class="card-panel deep-orange lighten-3">
```

Comment:
I think this block should come after the graphs and table of each benchmark since it is not relevant for most users / other fuzzer authors ?

Reply:
Do you mean it is only relevant for the author of the fuzzer that doesn't have enough samples? Will they see this if we put it somewhere hard to see?

Reply:
Yes i mean it is only relevant for that particular fuzzer author. I think since it is card and benchmark section is small, it is still very visible if put at end part of experiment section.

Reply:
isn't it useful for any reader of the report to make anomalies very visible? (if i recall correctly, one reason we wanted to add this was because we haven't noticed some fuzzers missing / having few trials in some experiments for a long time.)
how about moving it after the plots, but before the summary table? i tried putting it after the table, but there it can be confusing to decide which benchmark it belongs to, as it's right on top of the next benchmark's title. i think having it above the summary table makes more sense, because it also suggests expanding the table to check the exact sample numbers in the table. wdyt?